generators/app/templates/package.json

```diff
@@ -36,6 +38,7 @@
     "istanbul-combine": "^0.3.0",
     "json2yaml": "^1.1.0",
     "mocha": "^3.0.1",
+    "swagger-cli": "^1.0.0-beta.2",
```

Comment:
@brunogsa you can use the `swagger-tools` package instead of this one.
Take a look at [its docs](https://github.com/apigee-127/swagger-tools/blob/master/docs/CLI.md)